public/client.js

```diff
@@ -38,33 +38,31 @@ function gameSetup() {
   };
 }
 
+/**
+ * Chrome Android browser does not like it when
+ * 
+ * background image positions are set to 100%. capping this to 99.6%, which seems ok
+ * 
+ * @param {number} idx
+ * 
+ * @return {number}
+ */
+function backgroundPosition(idx) {
+  return (idx * 99.6) / 3;
+}
+
 // Position the image in the right place on each tile to reassemble it on the grid, and enable click events on the tiles
 function tileSetup() {
   const tileArray = document.querySelectorAll(".tile");
-  const gameArea = document.getElementById("game-area");
-
-  tileArray.forEach(function(tile, index) {
-    // inexplicably, Chrome Android browser does not like it when some background image positions are set to 100%.
-    // therefore capping this to 99.6%, which seems to display ok
-    const backgroundPositionX = ((index % 4) * 99.6) / 3;
```

Comment:
moved into the function. removed variable declaration because the `x` `y` value was only used once.